### ISSUE TITLE
Fatal error running `swiftlint rules` on Xcode 11 and MacOS Catalina

### ISSUE THREAD

**Issue by samyanez94** [realm/SwiftLint#2789]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

Running `swiftlint rules` is throwing a fatal error. I'm using Xcode 11.0 and I believe the issue might be related to this version of Xcode or MacOS Catalina.

##### Complete output when running SwiftLint, including the stack trace and command used

```bash
$ swiftlint rules
Fatal error: Loading sourcekitd.framework/Versions/A/sourcekitd failed: file /Users/marcelofabri/SwiftLint/.build/checkouts/SourceKitten/Source/SourceKittenFramework/library_wrapper.swift, line 66
zsh: illegal hardware instruction  Pods/SwiftLint/swiftlint rules
```

### Environment

* SwiftLint version? **0.33.0** 
* Installation method used? **Cocoapods**
* Configuration file:

```yml
excluded:
  - Pods

# Rule identifiers to exclude from running
disabled_rules:
  - trailing_whitespace

# Minimum line length
line_length: 240

# Minimum identifier length
identifier_name:
  min_length: 2
```

* Are you using [nested configurations](https://github.com/realm/SwiftLint#nested-configurations)? **No**
* Which Xcode version are you using? **Xcode 11.0 Beta 2**

**Comment by jpsim** [comment]
Do you also get a similar failure when running `swiftlint help`? Printing the result of `xcodebuild -version` would be useful too. Finally, could you please share the results of running these steps?

```shell
$ git clone https://github.com/realm/SwiftLint.git
$ cd SwiftLint
$ swift run swiftlint rules
```

Thanks for your help. I rely on this information to diagnose the issue because I don't have a machine running Catalina available to me and haven't been able to reproduce this issue with Mojave and Xcode 11 beta 2.

**Comment by samyanez94** [comment]
When running `xcodebuild -version` I noticed the following error: 

```
xcode-select: error: tool 'xcodebuild' requires Xcode, but active developer directory '/Library/Developer/CommandLineTools' is a command line tools instance
```

I referred to this [thread on Stack Overflow](https://stackoverflow.com/questions/17980759/xcode-select-active-developer-directory-error) and solved the issue. Afterwards, I was able to run `swiftlint rules`. I'm therefore closing the issue as it was an environment error. 

Thanks for the help! 😄
